components/layout_thread/lib.rs

```diff
@@ -869,7 +871,16 @@ impl LayoutThread {
         // Drop the root flow explicitly to avoid holding style data, such as
         // rule nodes.  The `Stylist` checks when it is dropped that all rule
         // nodes have been GCed, so we want drop anyone who holds them first.
-        debug!("layout: query has been waited: {}", self.layout_query_waiting_time);
+        let waiting_time_min = self.layout_query_waiting_time.minimum().expect("No waiting layout query.");
```

Comment:
@jdm it looks untidy here with dup-`expect`s, is there any way to refactor this?

Reply:
Not that I know of.